cou/utils/openstack.py

```diff
@@ -147,6 +147,12 @@
     "lunar": "antelope",
 }
 
+CEPH_RELEASES = [
+    "octopus",
```

Comment:
I think it's possible to encounter clouds using `natilus` if they are coming from bionic, so let's add to this list

Reply:
We are not letting people upgrade from bionic, so I don't think adding `nautilus` is necessary nor desirable

Reply:
I think it's easy to someone forget to do this post upgrade step of `require-osd-release` and have a focal cloud on natilus

Reply:
We are only using this CEPH_RELEASES list for checking whether the osd versions are in these range,  and IMO if their cloud is correct, they should never have OSD running `nautilus` on focal. If the operator starts an upgrade with `nautilus` value of `required_osd_release` but osd is at `octopus`, that value will be overwrite to `octopus` in the pre-upgrade step of ceph-mon.

Reply:
I see your point and will resolve this conversation, but as you said `required_osd_release` will overwrite to octopus in the pre-upgrade step of ceph-mon, so I think it's good to have it on `test_set_require_osd_release_option_different_releases`